src/sections/collection.js

```diff
@@ -0,0 +1,269 @@
+import "./styles/collection.scss";
+import { onDocumentReady } from "../utils/dom";
+import { addToCart } from "../utils/addToCart";
+
+const state = {
+  elements: {},
+};
+
+const cacheState = () => {
+  state.elements = {
+    sortBy: document.getElementById("SortBy"),
+    url: new URL(window.location.href),
+    collectionFilterCheckbox: document.querySelectorAll(
+      ".collectionFilter input[type='checkbox']",
+    ),
+    collectionFilterPrice: document.querySelectorAll(
+      ".filter-group-display__price-range input[type='number']",
+    ),
+    paginationItems: document.querySelectorAll("[data-pagination-item]"),
+    paginationLinks: document.querySelectorAll("[data-pagination-link]"),
+    collectionGrid: document.getElementById("CollectionGrid"),
+    collectionButton: document.querySelector(".collection__load--button"),
+    svgImage: `<svg width="10" height="8" viewBox="0 0 10 8" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
you are not using this svg anywhere. Next time if you need to render svg using JS, render it using Liquid and hide it and whenever you need show it using JS

Reply:
done, removed that svg image, didn't need it